src/ray/rpc/gcs_server/gcs_rpc_client.h

```diff
@@ -81,10 +73,7 @@ class GcsRpcClient {
   /// \param callback The callback function that handles reply from server.
   void UpdateActorInfo(const UpdateActorInfoRequest &request,
```

Comment:
Looks like all of these just call into the macro - is it possible to entirely avoid writing the boilerplate code?

Reply:
Done. Please take another look:)